tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -130,6 +137,16 @@ sub rke2_server_setup {
     # For network multus backend testing
     assert_script_run("sed -i '/ExecStart=/s/server\$/server --cni=multus,canal/' /etc/systemd/system/rke2-server.service");
 
+    # Setup cnv-bridge containernetworking plugin: one of the tests requires
+    # a newer version of the Linux bridge CNI:
+    #   https://github.com/kubevirt/kubevirt/commit/3fa7e2b67f9095d664e83eb1e13b587ab76b4950
+    assert_script_run("curl -LO https://github.com/containernetworking/plugins/releases/download/v1.1.1/cni-plugins-linux-amd64-v1.1.1.tgz");
+    assert_script_run("curl -LO https://github.com/containernetworking/plugins/releases/download/v1.1.1/cni-plugins-linux-amd64-v1.1.1.tgz.sha256");
+    assert_script_run("sha256sum --check cni-plugins-linux-amd64-v1.1.1.tgz.sha256");
+    assert_script_run("mkdir -p /opt/cni/bin");
+    assert_script_run("tar -xOf cni-plugins-linux-amd64-v1.1.1.tgz ./bridge > /opt/cni/bin/cnv-bridge");
+    assert_script_run("chmod +x /opt/cni/bin/cnv-bridge");
+
```

Comment:
This code block is quite similar with above in agent. Possible to wrap into some function then call them on both places?

Reply:
For this short code block, it's just some commands execution on both server and agent and no input/output parameters. So I think it's not necessary to encapsulate as a function.

Reply:
Function does not always need input or output, I think. Everyone is better to follow DRY -- do not repeat yourself, which is given in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md

Reply:
OK, updated.